x-pack/plugins/cases/docs/openapi/bundled.json

```
@@ -231,7 +231,8 @@
                   "type": "array",
                   "items": {
                     "type": "string"
-                  }
+                  },
+                  "maximum": 100
```

I think if the intention is to say that the array can contain (up to and including) 100 items, we ought to use "maxItems" instead, per example in https://swagger.io/docs/specification/data-models/data-types/#array
```suggestion
                  "maxItems": 100
```

---

Done